### DISCUSSION TITLE
Maximize shortcut not adapting to different monitor sizes

### DISCUSSION THREAD

**Discussion by adamjt** [rxhanson/RectanglePro-Community#348] [Q&A]
I have set the "Repeated commands" setting to "cycle through displays" and assigned a keyboard shortcut for the "Maximize" placement option. The window does move between displays as expected, however it keeps the horizontal size of the smaller display. Interestingly, it does seem to adapt vertically, so when I move a window to my larger external display, it fills roughly the left 80% of the screen. Any workarounds?

**Comment by rxhanson** [comment]
There is a quirk/bug in macOS where the commands to maximize on another display don't properly take effect on certain displays, and there are a few users that have reported this to me. I will have to test out your specific scenario and see if this is the issue here, or if there is a flaw in the repetitions logic somewhere along the way. I have a hunch it might be the former.

**Comment by adamjt** [comment]
@rxhanson Thanks! I'm using a MacBook Pro 16" with a Studio Display. This also happened on my older monitor which was a 4k LG. What can I do to help?
